Haaretz.js

```diff
@@ -0,0 +1,137 @@
+{
+	"translatorID": "d6f64d96-aa6f-4fd3-816f-bdef842c7088",
+	"label": "Haaretz",
+	"creator": "Eran Rosenthal",
+	"target": "^https?://www\\.haaretz\\.(co\\.il|com)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-08-15 15:42:07"
+}
+
+/**
+	Copyright (c) 2015 Eran Rosenthal
+	
+	This program is free software: you can redistribute it and/or
+	modify it under the terms of the GNU Affero General Public License
+	as published by the Free Software Foundation, either version 3 of
+	the License, or (at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+	Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public
+	License along with this program. If not, see
+	<http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	if (ZU.xpathText(doc, '//header/h1')) {
+		return 'newspaperArticle';
+	}
+}
+
+function doWeb(doc, url) {
+	var item = new Zotero.Item('newspaperArticle');
+	
+	var title = ZU.xpathText(doc, '//header/h1');
+	if(!title) {
+		title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	}
+	item.title = title;
+	item.url = url;
+	if(/haaretz\.com/.test(url)) {
+		item.publicationTitle = 'Haaretz';
+		item.language = 'en';
+	} else {
+		item.publicationTitle = 'הארץ';
+		item.language = 'he';
+	}
+
+	var abstract = ZU.xpathText(doc, '//header/p');
+	if (!abstract) abstract = ZU.xpathText(doc, '//meta[@property="og:description"]/@content');
+	item.abstractNote = abstract;
+
+	var authors = ZU.xpath(doc, '//address/a[@rel="author"]');
+	for(var i=0; i<authors.length; i++) {
+		item.creators.push(ZU.cleanAuthor(authors[i].textContent, 'author'));
+	}
+
+	item.date = ZU.xpathText(doc, '//meta[@property="vr:published_time"]/@content');
+	item.complete();
+}
+
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.haaretz.co.il/news/politics/.premium-1.2707588",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "פרשנות || נתניהו מעריך שיפסיד בקונגרס על הסכם הגרעין, ובונה על הרווח מכך",
+				"creators": [
+					{
+						"firstName": "חמי",
+						"lastName": "שלו",
+						"creatorType": "author"
+					}
+				],
+				"date": "Sat Aug 15 17:03:17 IDT 2015",
```

Comment:
We should clean this up a bit. The double year might screw up Zotero's parsing. I can make more concrete recommendations later